app/src/main/java/com/example/farmeraid/sign_in/SignInViewModel.kt

```diff
@@ -69,9 +69,10 @@ class SignInViewModel @Inject constructor(
                 Log.d("MESSAGE", "LOGGED IN")
                 userRepository.getUserId()?.let { Log.d("UserID", it) }
 
-                val farmId = userRepository.getFarmId()
+                val farmId = userRepository.getFarmId().toString()
+                Log.d("FARM ID", farmId.toString())
```

Comment:
Do you have to do `farmId.toString()` if you are already do toString() in line #72?